### PR TITLE
Refactor IconTooltip component to use appearance

### LINKED ISSUE #3192: Refactor IconTooltip component
`IconTooltip` should receive an `appearance` prop which can in turn be passed to the `Tooltip` component. Icon's `appearance` prop already has a `size` field with various standard options.

Was raised first here - https://github.com/JoinColony/colonyDapp/pull/3189#discussion_r806718282

### PR DESCRIPTION
**Pull request by arrenv**
## Description

The PR refactors the `IconTooltip` component to use an `appearance` prop, to maintain consistency with other components and provide standard sizes for icons. These, however can still be overridden with a custom class name.

**New stuff** ✨

* Adds `appearance` prop to `IconTooltip` component 
* Adds some standard sizes to the `IconTooltip` component 

**Deletions** ⚰️

* Removes `iconSize` prop

Resolves #3192

### CODE REVIEW

**Review by ArmandoGraterol — APPROVED** [review]
Nice refactor! It looks clean AF

File: src/modules/core/components/IconTooltip/IconTooltip.tsx [hunk 1]
```diff
@@ -9,8 +10,15 @@ import { getMainClasses } from '~utils/css';
...
+export interface Appearance {
+  size: 'tiny' | 'small' | 'medium' | 'large' | 'huge' | 'massive';
+  theme?: 'dark' | 'primary';
+}
+
 interface Props {
   icon: string;
+  /** Appearance object */
+  appearance?: Appearance;
   iconSize?: string;
```
Comment (dit7ya) [inline src/modules/core/components/IconTooltip/IconTooltip.tsx:22]: Is this being used anywhere anymore?

File: src/modules/core/components/IconTooltip/IconTooltip.tsx [hunk 2]
```diff
@@ -28,7 +36,7 @@ const displayName = 'IconTooltip';
 
 const IconTooltip = ({
   icon,
-  iconSize = styles.normalIcon,
+  appearance = { size: 'medium', theme: 'dark' },
```
Comment (dit7ya) [inline src/modules/core/components/IconTooltip/IconTooltip.tsx:39]: Does the `theme` field do anything?

**Review by dit7ya — APPROVED** [review]
Just have two minor questions, otherwise LGTM!

File: src/modules/core/components/IconTooltip/IconTooltip.tsx (hunk 2 above)
Comment (arrenv) [reply src/modules/core/components/IconTooltip/IconTooltip.tsx:39]: Thanks for the reminder, it is supposed to be used for the tooltip styling, I have updated it now.

**Review by chinins — COMMENTED** [review]
I know you have already merged but I wanted to have a look anyway. I think it looks much better and cleaner this way - nice work!

**Comment by arrenv** [comment]
@chinins thank you, feel free to take a look at this one https://github.com/JoinColony/colonyDapp/pull/3200, to fix this merge.

**Comment by chinins** [comment]
> @chinins thank you, feel free to take a look at this one #3200, to fix this merge.

Sure, it's already on my to review list :)

**Pull request merged**